source/ui/acquaman/AMGenericStepScanConfigurationView.cpp

```diff
@@ -106,8 +106,15 @@ AMGenericStepScanConfigurationView::AMGenericStepScanConfigurationView(AMGeneric
 
 	positionsBox->setLayout(positionLayout);
 
-	onAxisControlChoice1Changed();
-	onAxisControlChoice2Changed();
+	// Set initial spinbox values in cases where we don't yet have axes:
+	if(configuration_->scanAxes().count() == 0) {
```

Comment:
My only question with this is whether or not it properly displays and enables everything.  If someone had a 2D scan would both axes show up enabled?